### PR TITLE
Implement toast as a service

### PR DESCRIPTION
**Pull request by mydea**
This PR implements a `toast` service. It works as it did before, however it has the advantage that it can be used & injected like any other service. This makes it possible to simply inject it into other objects, e.g. models or services.

### CODE REVIEW

File: addon/services/toast.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        window.toastr[name](msg.toString(), title.toString(), options);
+    };
+};
+
+export default Ember.Service.extend({
+    success: proxyGenerator('success'),
+    info: proxyGenerator('info'),
+    warning: proxyGenerator('warning'),
+    error: proxyGenerator('error')
+});
```
Comment (knownasilya) [inline addon/services/toast.js:14]: Could you make the "tabs" here 2 spaces? Thanks.

**Comment by knownasilya** [comment]
Thanks! See my one comment above.

**Comment by mydea** [comment]
Sorry, this slipped through ;) Should be fixed now.

**Comment by knownasilya** [comment]
Published as 1.3.0

**Pull request merged**